java/test/jmri/jmrit/operations/trains/XmlTest.java

```diff
@@ -26,6 +28,68 @@
  * @version $Revision$
  */
 public class XmlTest extends OperationsTestCase {
+    
+    public void testFilePathNames() {
+        // test the build report path name
+        Assert.assertEquals("buildstatus", TrainManagerXml.BUILD_STATUS);
```

Comment:
Since TrainManagerXml.BUILD_STATUS is constant, this test is of no value. This is true of all other tests that assert that a constant is of a certain value.